wrappers/swift/Sample/main.swift

```diff
@@ -3,8 +3,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import ObjCModule // Required if Enums defined in ObjC headers are being used
+import OneDSSwift // Swift Package containing the wrappers over ObjC
+
 let props = EventProperties(name:"TestEvent")
 props.setProperty("PropName", withValue: ["Type":"SwiftWrappers"])
 props.setProperty("PropWithPII", withInt64Value: Int64(30), withPiiKind: ODWPiiKind.distinguishedName)
-print (props.properties)
-print (props.piiTags)
+print(props.properties())
```

Comment:
Can we have the sample project use the package manifest, please? It would be nice to see how projects will integrate this Swift wrapper. Do we need to move the sample project one level up?

Reply:
I will take that as a separate task and create a project, in examples folder. Planning to keep this swift file as a standalone example of usage.